Fix durability rendering

Michal-MK opened the pull request:
Before:
![https://i.imgur.com/SvafOQW.png](https://i.imgur.com/SvafOQW.png)

After:
![https://i.imgur.com/S7Dz6VW.png](https://i.imgur.com/S7Dz6VW.png)

Michal-MK committed:
Fixed durability rendering in the Request Table
- also moved the durability bar down by 2 pixels to cover less of the item (there is enough space for that)

Michal-MK committed:
Actual fix instead of hacking alpha to 1 ;)
- shifted ones to alpha channel bits

bziemons commented on common/logisticspipes/utils/gui/SimpleGraphics.java:
```diff
@@ -47,7 +47,6 @@ public static void drawHorizontalLine(int x1, int x2, int y, Color color, int th
 	 * @param y         the y-coordinate the line is on
 	 * @param color     the color, which the line will have
 	 * @param thickness the thickness, which the line will have
-	 * @see net.minecraft.client.gui.Gui#drawHorizontalLine(int, int, int, int)
```
was the function on Minecraft side removed or just moved?

Michal-MK replied:
I did not dig that deep into it

Michal-MK replied:
![image](https://user-images.githubusercontent.com/12874766/123678913-32b60780-d847-11eb-9d48-d5969ead90a1.png)

bziemons replied:
It doesn't really matter if it is inaccessible. It is a piece of information that shouldn't just be removed

bziemons commented on common/logisticspipes/utils/gui/SimpleGraphics.java:
```diff
@@ -262,10 +259,18 @@ public static void drawQuad(Tessellator tessellator, int x, int y, int width, in
+		float r = Color.getRed(color);
+		float g = Color.getGreen(color);
+		float b = Color.getBlue(color);
+		buf.pos(x, y, zLevel)
+				.color(r, g, b, a).endVertex();
+		buf.pos(x, y + height, zLevel)
+				.color(r, g, b, a).endVertex();
+		buf.pos(x + width, y + height, zLevel)
+				.color(r, g, b, a).endVertex();
+		buf.pos(x + width, y, zLevel)
+				.color(r, g, b, a).endVertex();
```
adding newlines before `.color` doesn't seem necessary and looks out of place

Michal-MK replied:
I like it better since the pos() has variable length, but if you want it on one line I will change it.

bziemons replied:
I am fine with it, if you put `.endVertex();` on a new line as well, since that makes is less random while improving readability

Michal-MK committed:
Re-added the inaccessible method references
- suppressed JavaDoc errors
- split color/endVertex but IDEA is dumb so it still looks off

Michal-MK commented:
The one I missed should be back ;)

bziemons commented:
Thank you :)
